mantidproject/mantid
# MaxEnt improvements

Adds some improvements to `MaxEnt`.

**To test:**
1. Run the following script:

```
Load(Filename=r'EMU00020884.nxs', OutputWorkspace='ws')
CropWorkspace(InputWorkspace='ws', OutputWorkspace='ws', XMin=0.17, XMax=4.5, EndWorkspaceIndex=0)
ws = RemoveExpDecay(InputWorkspace='ws')
ws = Rebin(InputWorkspace='ws', Params='0.016')
evolChi, evolAngle, image, data = MaxEnt(InputWorkspace='ws', A=0.0001, ChiTarget=300, MaxIterations=2500)
```
1. Make sure you can run the script without the `Rebin` step.
2. Inspect the output workspaces `data` and `image`. `data` should have (X) units of `Time` in microseconds, and `image` should be `Frequency` in MHz.
3. Modify the script to use a `DensityFactor` that will increase the number of points both in the image and reconstructed data:

```
Load(Filename=r'EMU00020884.nxs', OutputWorkspace='ws')
CropWorkspace(InputWorkspace='ws', OutputWorkspace='ws', XMin=0.17, XMax=4.5, EndWorkspaceIndex=0)
ws = RemoveExpDecay(InputWorkspace='ws')
evolChi, evolAngle, image, data = MaxEnt(InputWorkspace='ws', A=0.0001, ChiTarget=300, MaxIterations=5000, DensityFactor=2)
```
1. Review the new section in the documentation that explains this option ("Increasing the density of points in the image").

Fixes #15265.

[Release notes](http://www.mantidproject.org/index.php?title=Release_Notes_3_7_MuonAnalysis&action=history)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
